pkg/controllers/cloud_config_sync_controller_test.go

```diff
@@ -81,43 +81,57 @@ func makeInfraStatus(platform configv1.PlatformType) configv1.InfrastructureStat
 	}
 }
 
-func makeInfraCloudConfig() *corev1.ConfigMap {
+func makeInfraCloudConfig(platform configv1.PlatformType) *corev1.ConfigMap {
+	defaultConfig := `[Global]
+`
+
+	if platform == configv1.AzurePlatformType {
+		defaultConfig = defaultAzureConfig
+	}
+
 	return &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{
 		Name:      infraCloudConfName,
 		Namespace: OpenshiftConfigNamespace,
-	}, Data: map[string]string{infraCloudConfKey: defaultAzureConfig}}
+	}, Data: map[string]string{infraCloudConfKey: defaultConfig}}
 }
 
-func makeManagedCloudConfig() *corev1.ConfigMap {
+func makeManagedCloudConfig(platform configv1.PlatformType) *corev1.ConfigMap {
+	defaultConfig := `[Global]
```

Comment:
nit: Is this changing the meaning of default to be an aws config, rather than azure? 
I might be tempted to switch on platformtype here to make things explicit.

Reply:
Originally, all platforms used INI, most of the CCMs are still INI configured, Azure on the other hand, is the exception.

Some support INI or yaml, not AWS, not Azure 🤦

Reply:
TIL, and Oh, god 😞